offline-geocode/README.md

```diff
@@ -0,0 +1,72 @@
+# Offline geocode
+This sample demonstrates how you can geocode addresses to locations and reverse geocode locations to addresses when the device does not have network connectivity
+
+# How to use the sample
+The sample depends on basemap data to be located on the device. This includes installing a local tile map cache (tpk) to device as described below:
+
+ * Download San Francisco [tpk](http://www.arcgis.com/home/item.html?id=9a7e015149854c35b5eb94494f4aeb89) and [locator](https://www.arcgis.com/home/item.html?id=3109c0a575f547e6961c3ccc294bf67a) data from ArcGIS.
+ * Create the the sample data folder at the root folder on your device, /{device-externalstoragepath}/ArcGIS/samples/Offlinegeocode.
+ * Push the downloaded data contents from step 1 to your device:
+  * /{device-externalstoragepath}/ArcGIS/samples/Offlinegeocode/SanFrancisco.tpk
+  * /{device-externalstoragepath}/ArcGIS/samples/Offlinegeocode/Geocoding/SanFrancisco.loc
+  * /{device-externalstoragepath}/ArcGIS/samples/Offlinegeocode/Geocoding/SanFrancisco.locb
+  * /{device-externalstoragepath}/ArcGIS/samples/Offlinegeocode/Geocoding/SanFrancisco.lox
+  * /{device-externalstoragepath}/ArcGIS/samples/Offlinegeocode/Geocoding/SanFranciscoLocator.loc.xml
```

Comment:
Remove the additional **/Geocoding/** folder as sample does not work when putting locator data in the folder, should be in the folder above